### ISSUE TITLE
Incompatible with React 0.14: "Cannot assign to read only property 'id' of #<Object>"

### ISSUE THREAD

**Issue by EvHaus** [reactjs/react-a11y#79]
I've been migrating to React 0.14 and I'm starting to get this error from react-a11y when instantiating new components via `renderSubtreeIntoContainer`:

```
TypeError: Cannot assign to read only property 'id' of #<Object>
    at Object.React.createElement (webpack:///./~/react-a11y/dist/index.js?:171:14)
    at render (eval at executeCode (webpack:///./html/js/libs/ReactPlayground.js?), <anonymous>:23:16)
    at ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext (webpack:///./~/react/lib/ReactCompositeComponent.js?:546:34)
    at ReactCompositeComponentMixin._renderValidatedComponent (webpack:///./~/react/lib/ReactCompositeComponent.js?:566:32)
    at ReactPerf.measure.wrapper (webpack:///./~/react/lib/ReactPerf.js?:66:21)
    at ReactCompositeComponentMixin.mountComponent (webpack:///./~/react/lib/ReactCompositeComponent.js?:181:32)
    at ReactPerf.measure.wrapper [as mountComponent] (webpack:///./~/react/lib/ReactPerf.js?:66:21)
    at Object.ReactReconciler.mountComponent (webpack:///./~/react/lib/ReactReconciler.js?:37:35)
    at ReactDOMComponent.ReactMultiChild.Mixin.mountChildren (webpack:///./~/react/lib/ReactMultiChild.js?:187:44)
    at ReactDOMComponent.Mixin._createContentMarkup (webpack:///./~/react/lib/ReactDOMComponent.js?:498:32)
```

Any ideas what could be going wrong?

**Comment by globexdesigns** [comment]
I believe this is caused by the fact that react-a11y is trying to change component `props` which is no longer allowed in React 0.14.

**Comment by devgeeks** [comment]
:+1:

**Comment by devgeeks** [comment]
Bumping for React v0.14.0 being out now.  ;)

**Comment by gonsfx** [comment]
This has been known since July, React 0.14 is out for more than 2 weeks, there is a pull request #96 fixing this and nothing happened yet?

Y U NO ACCEPT PR?

**Comment by romeovs** [comment]
:+1: let's fix this!

**Comment by xjamundx** [comment]
I think this fix has been merged and fixed as of v0.2.7

**Comment by erin-doyle** [comment]
I don't believe this is an issue any longer.  I'm closing this for now but if someone is still having this problem please reply and I'll reopen.
